actors/miner/src/lib.rs

```diff
@@ -5672,6 +5515,7 @@ struct DataActivationOutput {
     pub unverified_space: BigInt,
     pub verified_space: BigInt,
     // None indicates either no deals or computation was not requested.
+    #[allow(dead_code)]
     pub unsealed_cid: Option<Cid>,
```

Comment:
could probably just remove this completely

Reply:
opportunities to remove or consolidate code would be one of the points of doing this; it'd be worth chasing `DataActivationOutput` through the code to see if it's involved in flows that only exist because of multiple paths which are now reduced to a single path, and from there working out if we can compress or de-complexity the flow.